### ISSUE TITLE
about training model evaluation

### ISSUE THREAD

**Issue by jiaoml1996** [MIC-DKFZ/nnUNet#35]
Hi,Fabian. During training, I find some information:

==============================================================

2019-08-05 21:29:11.999288: val loss (train=False): -0.4625
2019-08-05 21:29:12.180008: Val glob dc per class: [0.9191570267380906, 0.5505475351815083]
2019-08-05 21:29:13.074796: lr is now (scheduler) 0.0003
2019-08-05 21:29:13.161585: current best_val_eval_criterion_MA is 0.75620
2019-08-05 21:29:13.226852: current val_eval_criterion_MA is 0.7540
2019-08-05 21:29:13.259483: New best epoch (train loss MA): -0.4183
2019-08-05 21:29:13.300148: Patience: 0/50
2019-08-05 21:29:13.347112: This epoch took 1807.986493 s

================================================================
which information can indicate the quality of the model?  Val glob dc per class？ or current best_val_eval_criterion_MA?  And my understanding is that the higher the best_val_eval_criterion_MA , the better the model.

Thanks~

**Comment by FabianIsensee** [comment]
Hi,
you look at what you are interested in. That is the none of these measures :-) What you are really interested in cannot be computed during patch based training. `Val glob dc per class` is a good substitute measure though. But really, what you should be looking at is the validation dice once training is done. Use the other measures only to see if anything is happening at all
Best,
Fabian

**Comment by FabianIsensee** [comment]
Every measure in there has its own caveats. You can compare two trainings via their loss value IF the loss is the same (meaning you didn't change the loss function). You can also compare two trainings using `Val glob dc per class`, but keep in mind this is not the target metric as it is a global dice score, and not on a per-case basis.

**Comment by jiaoml1996** [comment]
I think I already understand, thank you~
